Ds3/Runtime/HttpHelper.cs

```diff
@@ -43,15 +53,15 @@ internal static class HttpHelper
         /// <returns></returns>
         public static string PercentEncodePath(string path)
         {
-            return CustomPercentEscaper.PercentEncode(path, UnreservedCharsPath);
+            return CustomPercentEscaper.PercentEncode(path, AllowedCharsPath);
         }
 
         /// <summary>
-        /// Implements percent encoding of a URI path as specified by RFC 3986 Section 2.1
-        /// http://tools.ietf.org/html/rfc3986#section-2.1
```

Comment:
Query params are specified in https://tools.ietf.org/html/rfc3986#section-3.4

Reply:
you need to look in Param section and not in the query because we are encoding the param and not the query